css/component_styles/getStarted.css

```diff
@@ -0,0 +1,89 @@
+#components {
+    margin-top: 5rem;
+}
+
+/* drawer css */
+.drawer {
+    background-color: var(--secondary-color);
+    padding: 1rem;
+}
+
+.drawer ul {
+    display: flex;
+    column-gap: 1rem;
+    overflow: auto;
+}
+
+::-webkit-scrollbar {
+    display: none;
+}
+
+.drawer ul li {
+    padding: 0.5rem;
```

Comment:
Keep small units in `px` rather than `rem`

Reply:
yeah ! thanks for the suggestion, i did changed that